includes/admin/pages.php

```diff
@@ -228,43 +231,183 @@ public function html() {
+										></path>
+								</svg>
+								<svg class="h-6 transition-all duration-100 fill-gray-400  cursor-pointer"
+									viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg"
+									xmlns:xlink="http://www.w3.org/1999/xlink">
+									<path
+										d="M575.852903 115.426402L661.092435 288.054362c10.130509 20.465674 29.675227 34.689317 52.289797 37.963825l190.433097 27.62866c56.996902 8.288598 79.7138 78.281203 38.475467 118.496253l-137.836314 134.35715c-16.372539 15.963226-23.84251 38.987109-19.954032 61.49935l32.540421 189.716799c9.721195 56.792245-49.833916 100.077146-100.793444 73.267113L545.870691 841.446188a69.491196 69.491196 0 0 0-64.67153 0l-170.376737 89.537324c-50.959528 26.810033-110.51464-16.474868-100.793444-73.267113L242.569401 667.9996c3.888478-22.512241-3.581493-45.536125-19.954032-61.49935L84.779055 472.245428c-41.238333-40.215049-18.521435-110.207655 38.475467-118.496252l190.433097-27.62866c22.61457-3.274508 42.159288-17.498151 52.289797-37.963826L451.319277 115.426402c25.479764-51.675827 99.053862-51.675827 124.533626 0z"></path>
+								</svg>
+							</div>
+						
+						<script>
+							const svgs = document.querySelector('.rating').children;
+							for(let i = 0;i<svgs.length;i++){ 
+								svgs[i].onclick = ()=>{
+									for(let j = 0;j<=i;j++){
+										svgs[j].classList.add("fill-yellow-200"); // this class should be added to whitelist while in production mode
+									}
+									for(let k = i + 1;k<svgs.length;k++){
+										svgs[k].classList.remove("fill-yellow-200"); // this class should be added to whitelist while in production mode
+									}
+								}
+							}
+						</script>
+						</div> 
+						<button type="button" class="mt-[20px] m-auto flex justify-center items-center w-[405px] h-[40px] bg-sc_green-200 text-white text-xl font-medium rounded-md">
+							Rate Now							
+						</button>
+						<div class=" mt-[25px] text-center text-base text-sc_blue-200 hover:underline">
+							<a href="">See All Rating 3.8 Out of 5 Stars</a>
+						</div>
+					</div>
+					<?php
+					echo '</div>';
+				
 				
+				?>
+				<div class="bg-sc_green-100 h-[399] w-[1302] ml-[102] mt-[41] flex max-xl:ml-[0px]">
+					<div class="w-[654] flex relative ">
+						<div class="ml-[37px] mt-[18px]">
+							<img src="<?php echo $admin_image_about_path . '/image1.png'; ?>" />
+						</div>
+						<div class="absolute right-0 top-28">
+							<img src="<?php echo $admin_image_about_path . '/Group 12.png'; ?>" />
```

Comment:
please rename all the images ad per their sections don't use this group and vector and if you need to add number in the image name then please use - or _ rather than space.

Reply:
i had change the name of all the images.